src/app/src/App.jsx

```diff
@@ -22,38 +21,29 @@ import Maintenance from './containers/Maintenance';
 import 'react-toastify/dist/ReactToastify.css'; // eslint-disable-line import/first
 import './App.css';
 
+import { sessionLogin } from './actions/auth';
+
+import { userPropType } from './util/propTypes';
+
 const styles = {
     root: {
         flexGrow: 1,
     },
 };
 
-const mapStateToProps = state => ({
-    user: state.user,
-});
-
-const mapDispatchToProps = dispatch => ({
-    actions: bindActionCreators(userActions, dispatch),
-});
-
 class App extends Component {
-    componentWillMount() {
-        if (!this.props.user.betaAccess) {
-            this.props.actions.checkAccess();
-        }
-
-        this.props.actions.loadUser();
+    componentDidMount() {
+        return this.props.logIn();
     }
 
-    shouldComponentUpdate(nextProps) {
-        return (
-            nextProps.user.loaded !== this.props.user.loaded ||
-            nextProps.user.betaAccess !== this.props.user.betaAccess
-        );
-    }
+    // shouldComponentUpdate(nextProps) {
+    //     return (
+    //         nextProps.user.loaded !== this.props.user.loaded ||
+    //         nextProps.user.betaAccess !== this.props.user.betaAccess
+    //     );
+    // }
```

Comment:
This part can probably just be deleted.

Likewise I realized that it might be that the legacy User reducer &c is no longer used in the app. I'll check and if so just delete it altogether.

Reply:
Deleted all of that in 
a0ecdb7 

I left some placeholders for Contribute.jsx and will make an issue for adjusting that component.

Reply:
See https://github.com/open-apparel-registry/open-apparel-registry/issues/101